client/lib/site/jetpack.js

```diff
@@ -38,7 +38,6 @@ JetpackSite.prototype.updateComputedAttributes = function() {
 	this.canUpdateFiles = SiteUtils.canUpdateFiles( this );
 	this.canAutoupdateFiles = SiteUtils.canAutoupdateFiles( this );
 	this.hasJetpackMenus = versionCompare( this.options.jetpack_version, '3.5-alpha' ) >= 0;
-	this.hasJetpackThemes = versionCompare( this.options.jetpack_version, '3.7-beta' ) >= 0;
```

Comment:
This is obsolete, `site.hasJetpackThemes` isn't used anywhere anymore. (Try grepping for `hasJetpackThemes`.)